packages/gitmoji-changelog-cli/src/cli.js

```diff
@@ -1,14 +1,32 @@
 const fs = require('fs')
 
+const libnpm = require('libnpm')
+const semverCompare = require('semver-compare')
 const { generateChangelog, logger } = require('@gitmoji-changelog/core')
 const { buildMarkdownFile } = require('@gitmoji-changelog/markdown')
 
 const pkg = require('../package.json')
 
+async function getLatestVersion() {
+  const watchdog = new Promise(resolve => setTimeout(resolve, 500, { version: pkg.version }))
+  const request = libnpm.manifest('gitmoji-changelog@latest')
+
+  const { version } = await Promise.race([watchdog, request])
+
+  return version
+}
+
 async function main(options = {}) {
   logger.start(`gitmoji-changelog v${pkg.version}`)
   logger.info(`${options.mode} ${options.output}`)
 
+  try {
+    const latestVersion = await getLatestVersion()
+    if (semverCompare(latestVersion, pkg.version) > 0) {
+      logger.warn(`You got an outdated version of gitmoji-changelog, please update! (yours: ${pkg.version}, latest: ${latestVersion})`)
```

Comment:
Maybe we could propose the npm command line, then the user just have to copy it:
```js
Just do the following npm command to update it:
npm install -g gitmoji-changelog@latest
```